lib/validators/custom-styles-validator.ts

```diff
@@ -0,0 +1,33 @@
+import { ComponentSet } from '../models';
+import { Validator } from './validator';
+import * as path from 'path';
+
+export class CustomStylesValidator extends Validator {
+    constructor(error: (errorMessage: string) => false, definition: ComponentSet, private filePaths: Set<string>) {
+        super(error, definition);
+    }
+
+    async validate(): Promise<void> {
+        this.checkCustomStylesFolder();
+
+        if (!this.componentSet.customStyles) {
+            return;
+        }
+
+        this.componentSet.customStyles.forEach((customStyle) => {
```

Comment:
This part should be in a separate `validate` function so that it explains by its name what it is doing.

Reply:
I just added the new function.